redo/__init__.py

```diff
@@ -56,21 +56,32 @@ def retrier(attempts=5, sleeptime=10, max_sleeptime=300, sleepscale=1.5, jitter=
         ...     print "max tries hit"
         max tries hit
     """
+    if jitter > sleeptime:
+        # To prevent negative sleep times
+        raise Exception('jitter (%i) must be less than sleep time (%i)', jitter, sleeptime)
+
+    sleeptime_real = sleeptime
     for _ in range(attempts):
         log.debug("attempt %i/%i", _ + 1, attempts)
-        yield
+
+        yield sleeptime_real
```

Comment:
What's the reasoning behind yielding the sleeptime? If anything, I would expect this to yield the attempt number.